### DISCUSSION TITLE
关于本项目后期发展事宜

### DISCUSSION THREAD

**Discussion by ronggang** [ronggang/PT-Plugin-Plus#1200] [General]
不知不觉项目自重启至今已有4个年头，感谢各位4年来的支持，特别感谢 @Rhilip @ted423 及其他参与者的付出！

因这两年本人生活和工作发生很大变化，没有精力投入开源项目，今后可能也无法在此项目上投入更多精力，在此和各位说声抱歉！

为了不影响各位的后续使用，近期我会尝试将项目转至 https://github.com/pt-plugins 组织下，并尝试发布本阶段最后一个版本。

后期项目由 @Rhilip @ted423 及其他参与者主导维护，本人将不再主导项目发展方向，希望项目发展越来越好。

栽培者 2022.12.10 晚

-----

- 短期内更新及版本发布见说明： [#1200 (discussioncomment-4377703)](https://github.com/pt-plugins/PT-Plugin-Plus/discussions/1200#discussioncomment-4377703)

**Comment by luckiestone** [comment]
谢谢，辛苦啦！

**Reply by laniv** [reply]
谢谢，辛苦啦！

**Comment by fzlins** [comment]
谢谢，辛苦了！

**Comment by chuanjin-su** [comment]
谢谢，辛苦了！（Edge商店版本是不是也可以转交呀，这样商店版本也可以保持更新hhh

**Reply by ronggang** [reply]
会尝试使用API自动发布功能。

**Reply by chuanjin-su** [reply]
好的，期待商店版本的后续更新~
再次感谢大佬的付出！

**Comment by jinglekang** [comment]
感谢大佬的付出！

**Comment by frankxff** [comment]
谢谢大佬，愿走过半生，归来仍是少年！

**Comment by LongLights** [comment]
感谢大佬，辛苦啦！

**Comment by AmosInk** [comment]
感谢大佬，辛苦了

**Comment by se7enmuting** [comment]
大佬，辛苦了，感谢这个插件给我带来的便利

**Comment by Jadaff** [comment]
感谢大佬，辛苦了

**Comment by sdshdv** [comment]
谢谢大佬多年的付出

**Comment by chixinzei** [comment]
谢谢，辛苦了！

**Comment by odemon0** [comment]
谢谢，辛苦了！

**Comment by du-zhibin** [comment]
感谢多年的无私付出和精彩陪伴，辛苦了。

**Comment by RUANRUI1995** [comment]
感谢大佬无私付出！

**Comment by allaready** [comment]
感谢多年的无私付出和精彩陪伴，辛苦了。

**Comment by 404reposter** [comment]
感谢

**Comment by mc275** [comment]
感谢，辛苦了

**Comment by gZDJCVcH** [comment]
感谢，辛苦了

**Comment by iwinstar1980** [comment]
谢谢，非常感谢

**Comment by Rhilip** [comment]
关于后续的一些想法：
1. 1.6 将作为LTS版本支持到 `next版本 正式release`。但以2022年12月底作为新功能接收的节点，之后到LTS结束前，1.6仅作站点适配的更新。
2. 支持期内，1.6的版本编号改为四位格式 `1.6.<release-version>.<patch-version>` 。每次commit仍由action构建action版。所有git pull（以rebase形式）和git commit先并入dev分支，积累后以merge形式并入master分支。发布分为pre-release和release两部分。
	2.1 pre-release，基于dev分支，周期改为每半月，由action自动构建。
	2.2 release，基于master分支，根据git commit情况确定，初定每季度一次。
	2.3 两者pre-release和release的自动更新走不同通道，但crxid相同。（这意味着两个版本不能同时安装使用）
3. 商店部分。由于①以往的审核问题；②目前仅在edge商店上架，且商店在推MV3（意味着edge和chrome商店不能上传新的MV2插件）；③原先由ronggang账号发布，需要更换账号发布 等情况。除目前ronggang尝试发布的1.6.0版本外，后续将不会再在商店尝试上架。

**Reply by Rhilip** [reply]
对于2.3部分，从1.6.0之后的文件update_url更新为：

## chromium系列（包含 Chrome、Edge、以及国产等浏览器） **双通道**
- pre-release的走 https://pt-plugins.github.io/PT-Plugin-Plus/update/canary.xml
- release的走 https://pt-plugins.github.io/PT-Plugin-Plus/update/stable.xml

## Firefox **单通道**
走 https://pt-plugins.github.io/PT-Plugin-Plus/update/firefox.json

**Reply by 404reposter** [reply]
您好提个建议，现有user等级有四种格式：1.User师弟 2.(未烧)User 3. 列兵 4.加菲猫 User，希望后续能统一格式，谢谢

**Reply by zwjie12** [reply]
> 对于2.3部分，从1.6.0之后的文件update_url更新为：
> 
> ## chromium系列（包含 Chrome、Edge、以及国产等浏览器） **双通道**
> * pre-release的走 https://pt-plugins.github.io/PT-Plugin-Plus/update/canary.xml
> * release的走 https://pt-plugins.github.io/PT-Plugin-Plus/update/stable.xml
> 
> ## Firefox **单通道**
> 走 https://pt-plugins.github.io/PT-Plugin-Plus/update/firefox.json

请问这个更新地址该怎么使用呢？

**Comment by Heathcliff-third-space** [comment]
感谢你的付出，辛苦了，祝生活愉快~

**Comment by 2427768286** [comment]
感谢 辛苦了~

**Comment by fallenyy** [comment]
感谢大佬

**Comment by 619954559** [comment]
感谢大佬 您辛苦了！
遥祝生活、工作顺利。

**Comment by toniexly** [comment]
salute

**Comment by prospectalien** [comment]
辛苦啦，祝您前程似锦

**Comment by pengkaiwei** [comment]
Tks !

**Comment by xmtrockET** [comment]
感谢大佬，辛苦了~

**Comment by Tohrusky** [comment]
辛苦了，大佬！
希望后续能有更多的人参与开发维护吧

**Comment by Deezhui** [comment]
感谢大佬的付出

**Comment by yalidabuda** [comment]
感谢各位大佬！

**Comment by xtxgame** [comment]
虽然没用过，但是每一个开源大佬都值得尊敬，谢谢你

**Comment by luoji12103** [comment]
谢谢付出，辛苦了

**Comment by dayunnas** [comment]
大佬幸苦了

**Comment by mboyka** [comment]
非常好用，感谢大佬的无私付出

**Comment by roamergu** [comment]
感谢大佬的工作~祝一切顺利！

**Comment by lwsbluesky** [comment]
开源社区的荣耀。感谢一路的付出。

**Comment by GiovanniLiu** [comment]
感谢倾情奉献

**Comment by xcpanda** [comment]
您辛苦了! 感谢大佬的付出

**Comment by byiceb** [comment]
大佬辛苦了，祝一切顺利！

**Comment by YuLianChen** [comment]
感谢大佬坚持了这么多年

**Comment by ttxhxz** [comment]
谢谢，辛苦啦！

**Comment by justmagic7211** [comment]
谢谢，辛苦啦！

**Comment by leverule** [comment]
感谢！祝大佬生活美满幸福

**Comment by cpf7115** [comment]
感谢大佬，辛苦啦！

**Comment by kirincc** [comment]
感谢大佬，祝前程似锦

**Comment by dzplus** [comment]
感谢大佬的付出 通过这个插件学到很多知识 祝大佬开启人生新旅程

**Comment by pwbty** [comment]
感谢多年付出!!

**Comment by liufuyou** [comment]
辛苦了，大佬
感谢多年来的付出!!

**Comment by 777Arch** [comment]
辛苦，感谢，祝更好！

**Comment by kemisstep** [comment]
大佬辛苦了，祝一切顺利！

**Comment by kemisstep** [comment]
大佬辛苦了，祝一切顺利！

**Comment by 313255924** [comment]
感谢大佬，辛苦啦

**Comment by ruchid123123** [comment]
请问会尝试在chrome重新上架吗

**Comment by jie8545** [comment]
辛苦啦 大白

**Comment by Leon19960120** [comment]
谢谢你！

**Comment by zoro1ee** [comment]
谢谢大会，辛苦了

**Comment by yunnl** [comment]
感谢，祝身体健康，工作顺利

**Comment by yaolongda** [comment]
感谢大佬之前的付出~辛苦啦

**Comment by seasea3216** [comment]
早日康复
